src/components/graph-view.js

```diff
@@ -1071,7 +1054,17 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
         }
 
         if (onUpdateNode) {
-          onUpdateNode(nodeMap.node);
+          let updatedNodes = null;
+
+          if (selectedNodes?.size > 0) {
+            updatedNodes = new Map(
+              Array.from(selectedNodes.keys()).map(selectedNodeID => {
+                return [selectedNodeID, this.getNodeById(selectedNodeID).node];
```

Comment:
Would it make more sense to copy existing map, then only update node that changed?
```
if (selectedNodes?.size > 0) {
  updatedNodes = new Map(selectedNodes);
  updatedNodes.set(nodeId, nodeMap.node);
}
```

Reply:
No, because we need to update all selected nodes with the same relative information. The external website handles that change, we just needed to pass them the map.

Reply:
So is this really `onUpdateNodes` (plural not singular)?